src/System.CommandLine/Argument.cs

```diff
@@ -146,8 +165,17 @@ private protected override string DefaultName
 
         internal List<ValidateSymbol<ArgumentResult>> Validators { get; } = new List<ValidateSymbol<ArgumentResult>>();
 
+        /// <summary>
+        /// Adds a custom <see cref="ValidateSymbol{T}(ArgumentResult)"/> to the argument. Validators can be used
+        /// to create custom validation logic.
+        /// </summary>
+        /// <param name="validate">The delegate to validate the symbols during parsing.</param>
         public void AddValidator(ValidateSymbol<ArgumentResult> validator) => Validators.Add(validator);
 
+        /// <summary>
+        /// Gets the default value for the argument.
+        /// </summary>
+        /// <returns>Returns the default value for the argument, if defined. Null otherwise.</returns>
```

Comment:
I believe by default it returns the default for the `ArgumentType`, not `null`. For example it might return `0` if `ArgumentType` is `int`.

Reply:
The "Null otherwise" is definitely incorrect, good catch.

However, looking at the code again, it will throw if you haven't set a default value. If you set a default value (or factory), then it doesn't need to find the default value for the type.

I can't see an obvious scenario where you can make this method return the default value for the ArgumentType type. Help!